doc/lua_api.txt

```diff
@@ -2470,6 +2470,7 @@ This is basically a reference to a C++ `ServerActiveObject`
 
 ##### Player-only (no-op for other objects)
 * `get_player_name()`: returns `""` if is not a player
+* `get_player_velocity()`: returns `nil` if is not a player otherwise a table {x, y, z} representing the player's instant velocity
```

Comment:
A better phrasing would be "returns `nil` if is not a player, otherwise a table {x, y, z}, representing the player's current velocity"